sig-release/charter.md

```diff
@@ -0,0 +1,85 @@
+# SIG Release Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+- Production of Kubernetes releases on a reliable schedule
+- Ensure there is a consistent group of community members in place to support the release process across time
+- Provide guidance and tooling to facilitate the production of automated releases
+- Serve as a tightly integrated partner with other SIGs to empower SIGs to integrate their repositories into the release process
+
+### In scope
+
```

Comment:
Typically I would expect to see Code and Binaries enumerated here.  Going by sigs.yaml you apparently own hyperkube, `anago`, and docs for the release process.

- Do you own package definitions that would be used to build .deb's, .rpm's or other packages, or are those considered downstream?
- Do you own any of the files involved in building kubernetes? Things like k/k/build/ or k/k's Makefiles?
- Do you own any jobs related to continuously building kubernetes, or are you solely responsible for manually produced builds?